src/file.js

```diff
@@ -1827,6 +1871,41 @@ class File extends common.ServiceObject {
       });
   }
 
+  /**
+   * Hold this file from its bucket's retention period configuration.
+   *
+   * By default, this will set an event-based hold. This is a way to retain
+   * objects until an event occurs, which is signified by the hold's release
+   * (i.e. this value is set to false (see @{link File#release})). After being
+   * released, this object will be subject to bucket-level retention (if any).
+   *
+   * Alternatively, you may set a temporary hold. This will follow the same
+   * behavior as an event-based hold, with the exception that the bucket's
+   * retention policy will not renew for this file from the time the hold is
+   * released.
+   *
+   * @param {object} [options] Configuration object.
+   * @param {boolean} [options.temporary=false] - Set a temporary hold.
```

Comment:
Should be set to true in documentation, I think?

Reply:
By default, it is set to false in our code:

```js
if (options.temporary) {
  metadata.temporaryHold = true;
} else {
  metadata.eventBasedHold = true;
}
```

Should we flip it, so that by default, a "file.hold()" creates a temporary hold, as opposed to an event-based hold?

Reply:
My mistake please ignore*